protocol-dashboard/src/components/UniqueUsersChart/UniqueUsersChart.tsx

```diff
@@ -29,14 +45,17 @@ const UniqueUsersChart: React.FC<UniqueUsersChartProps> = () => {
   }
   return (
     <LineChart
+      topNumber={topNumber}
       title="Unique Users"
       tooltipTitle="Users"
       data={data}
       labels={labels}
       selection={bucket}
       error={error}
-      options={[Bucket.ALL_TIME, Bucket.MONTH, Bucket.WEEK]}
-      onSelectOption={(option: string) => setBucket(option as Bucket)}
+      options={[Bucket.ALL_TIME, Bucket.YEAR, Bucket.MONTH, Bucket.WEEK]}
+      onSelectOption={(option: string) =>
```

Comment:
if onSelectOption already specifies option type, we dont need string, if it allows "any" type you can prob just do `onSelectOption={setBucket}`?